schemas/2017-03-01/Microsoft.ApiManagement.json

```diff
@@ -346,6 +349,53 @@
       ],
       "description": "Microsoft.ApiManagement/service/apis/policies"
     },
+    "service_apis_schemas": {
```

Comment:
>service_apis_schemas [](start = 5, length = 20)

do you need to add this to deploymentTemplate.json?

Reply:
#fixed, added test for resource